channels/ni.m3u

```diff
@@ -7,9 +11,13 @@ https://cootv.cootel.com.ni:443/streams/d/SSLCH11-SD-GZN/playlist.m3u8
 https://5ca9af4645e15.streamlock.net/canal12/videocanal12/playlist.m3u8
 #EXTINF:-1 tvg-id="Canal12.ni" tvg-name="Canal 12" tvg-country="NI" tvg-language="" tvg-logo="https://vignette.wikia.nocookie.net/logopedia/images/6/6a/Nicavision1993.png/revision/latest?cb=20180426154048" group-title="",Canal 12
 https://5ca9af4645e15.streamlock.net:443/canal12/videocanal12/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal12.ni" tvg-name="Canal 12" tvg-country="NI" tvg-language="" tvg-logo="https://vignette.wikia.nocookie.net/logopedia/images/6/6a/Nicavision1993.png/revision/latest?cb=20180426154048" group-title="",Canal 12
+http://209.91.213.10:8088/play/a00g
 #EXTINF:-1 tvg-id="Canal2.ni" tvg-name="Canal 2" tvg-country="NI" tvg-language="" tvg-logo="" group-title="",Canal 2
 https://cootv.cootel.com.ni/streams/d/SSLCH02-HD-YON/playlist.m3u8
 #EXTINF:-1 tvg-id="CANAL9.ni" tvg-name="CANAL 9" tvg-country="NI" tvg-language="" tvg-logo="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTawbZLCedJzRX06CAyU-jX37n9iyTxMVcIfg&usqp=CAU" group-title="",CANAL 9
 https://cootv.cootel.com.ni:443/streams/d/SSLCH09-HD-CVS/playlist.m3u8
 #EXTINF:-1 tvg-id="VIVANICARAGUACANAL13.ni" tvg-name="VIVA NICARAGUA CANAL 13" tvg-country="NI" tvg-language="" tvg-logo="" group-title="",VIVA NICARAGUA CANAL 13
 https://60417ddeaf0d9.streamlock.net/ch13/videoch13/chunklist_w1505653761_DVR.m3u8
```

Comment:
Streamlock link died. Were showing 8 sec chunk anyways

Reply:
Ah yeah, old link this one. Just try by only putting playlist.m3u8 at the end.